### PR TITLE
Add custom_captures config key to set highlights for custom queries

### PR DESCRIPTION
**Pull request by theHamsta**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -98,6 +98,10 @@ require'nvim-treesitter.configs'.setup {
     highlight = {
         enable = true,                    -- false will disable the whole extension
         disable = { 'c', 'rust' },        -- list of language that will be disabled
+        custom_captures = {              -- mapping of user defined captures to highlight groups
+          ----- highlight own capture @foo.bar with highlight group "Identifier", see :h nvim-treesitter-query-extensions
+          -- ["foo.bar"] = "Identifier"
```
Comment (vigoux) [inline README.md:103]: I am a bit touchy here, but can we make that the same as above ?

File: doc/nvim-treesitter.txt [hunk 2]
```diff
@@ -33,6 +33,11 @@ By default, everything is disabled. To enable support for features, in your `ini
     highlight = {
         enable = true,                    -- false will disable the whole extension
         disable = { 'c', 'rust' },        -- list of language that will be disabled
+        custom_captures = {              -- mapping of user defined captures to highlight groups
+          ----- highlight own capture @foo.bar with highlight group "Identifier",
+	  ----- see :h nvim-treesitter-query-extensions
+          -- ["foo.bar"] = "Identifier"
```
Comment (vigoux) [inline doc/nvim-treesitter.txt:39]: Same here ?

File: README.md (hunk 1 above)
Comment (theHamsta) [reply README.md:103]: Are you talking about the formatting?

Comment (vigoux) [reply README.md]: No just uncommenting this, in the end, this is just an example config.

**Pull request merged**